development/tools/cmd/gcrcleaner/README.md

```diff
@@ -0,0 +1,46 @@
+# GCR cleaner
+
+## Overview
+
+This command finds and removes old GCR images created by jobs in the Google Cloud Platform (GCP) project.
+
+There are three conditions used to find images for removal:
+- The repository name pattern is not on the ignored list.
+- The `creationTimestamp` value of the images, that is used to find addresses, exists at least for a preconfigured number of hours.
+
+GCR images that meet these conditions are subject to removal.
+
+## Usage
+
+For safety reasons, the dry-run mode is the default one.
+To run it, use:
```

Comment:
Mind that this is rendered as one line. To render it as two actually separate lines, you must either add two spaces at the end of the first line, or add a newline between them.